frontends/bmd/src/pages/__snapshots__/print_page.test.tsx.snap

```diff
@@ -1890,7 +1890,7 @@ exports[`renders PrintPage without votes and no seal 1`] = `
h7v1H0zM8 6h1v1H8zM10 6h1v1H10zM12 6h1v1H12zM14 6h1v1H14zM16 6h1v1H16zM18 6h1v1H18zM20 6h1v1H20zM22 6h1v1H22zM24 6h1v1H24zM26,6 h7v1H26zM8 7h4v1H8zM13 7h1v1H13zM17 7h4v1H17zM22 7h1v1H22zM24 7h1v1H24zM5 8h2v1H5zM9 8h1v1H9zM11 8h1v1H11zM15 8h1v1H15zM17 8h6v1H17zM24 8h1v1H24zM26 8h1v1H26zM28 8h1v1H28zM30 8h1v1H30zM32,8 h1v1H32zM0 9h3v1H0zM5 9h1v1H5zM7 9h5v1H7zM13 9h2v1H13zM16 9h7v1H16zM26 9h3v1H26zM30 9h2v1H30zM1 10h3v1H1zM5 10h4v1H5zM10 10h1v1H10zM12 10h1v1H12zM18 10h2v1H18zM22 10h1v1H22zM25 10h2v1H25zM28 10h4v1H28zM1 11h1v1H1zM11 11h4v1H11zM20 11h2v1H20zM23 11h1v1H23zM25 11h1v1H25zM27 11h1v1H27zM29,11 h4v1H29zM1 12h8v1H1zM10 12h2v1H10zM14 12h3v1H14zM18 12h1v1H18zM21 12h1v1H21zM23 12h1v1H23zM26,12 h7v1H26zM0 13h1v1H0zM3 13h2v1H3zM7 13h1v1H7zM9 13h5v1H9zM18 13h1v1H18zM23 13h7v1H23zM31 13h1v1H31zM1 14h1v1H1zM4 14h3v1H4zM8 14h1v1H8zM12 14h3v1H12zM17 14h1v1H17zM21 14h2v1H21zM25 14h1v1H25zM27 14h5v1H27zM0 15h2v1H0zM7 15h1v1H7zM10 15h1v1H10zM12 15h1v1H12zM15 15h6v1H15zM22 15h1v1H22zM27 15h1v1H27zM30 15h1v1H30zM32,15 h1v1H32zM1 16h2v1H1zM5 16h2v1H5zM8 16h1v1H8zM10 16h1v1H10zM12 16h1v1H12zM15 16h3v1H15zM25 16h1v1H25zM27 16h1v1H27zM29 16h1v1H29zM31,16 h2v1H31zM5 17h1v1H5zM9 17h1v1H9zM11 17h1v1H11zM16 17h2v1H16zM19 17h2v1H19zM22 17h3v1H22zM26 17h2v1H26zM29,17 h4v1H29zM0 18h2v1H0zM3 18h4v1H3zM8 18h1v1H8zM11 18h4v1H11zM16 18h1v1H16zM19 18h1v1H19zM21 18h1v1H21zM23 18h7v1H23zM31,18 h2v1H31zM0 19h2v1H0zM3 19h1v1H3zM5 19h1v1H5zM8 19h3v1H8zM12 19h2v1H12zM15 19h1v1H15zM25 19h1v1H25zM27 19h1v1H27zM29 19h3v1H29zM0 20h1v1H0zM2 20h2v1H2zM5 20h2v1H5zM8 20h1v1H8zM10 20h2v1H10zM13 20h1v1H13zM18 20h1v1H18zM20 20h1v1H20zM22 20h1v1H22zM24 20h1v1H24zM27 20h1v1H27zM0 21h3v1H0zM7 21h1v1H7zM9 21h5v1H9zM15 21h1v1H15zM18 21h2v1H18zM25 21h4v1H25zM32,21 h1v1H32zM0 22h1v1H0zM5 22h2v1H5zM10 22h2v1H10zM13 22h2v1H13zM17 22h2v1H17zM20 22h2v1H20zM23 22h2v1H23zM28 22h1v1H28zM30 22h2v1H30zM0 23h1v1H0zM2 23h1v1H2zM4 23h2v1H4zM10 23h1v1H10zM13 23h1v1H13zM15 23h1v1H15zM17 23h1v1H17zM19 23h1v1H19zM22 23h2v1H22zM25 23h3v1H25zM29 23h2v1H29zM32,23 h1v1H32zM0 24h2v1H0zM3 24h1v1H3zM6 24h3v1H6zM14 24h2v1H14zM18 24h1v1H18zM20 24h2v1H20zM24 24h5v1H24zM31 24h1v1H31zM8 25h1v1H8zM14 25h2v1H14zM18 25h1v1H18zM24 25h1v1H24zM28 25h2v1H28zM32,25 h1v1H32zM0 26h7v1H0zM9 26h1v1H9zM12 26h4v1H12zM17 26h1v1H17zM20 26h2v1H20zM23 26h2v1H23zM26 26h1v1H26zM28 26h4v1H28zM0 27h1v1H0zM6 27h1v1H6zM8 27h3v1H8zM12 27h1v1H12zM15 27h1v1H15zM18 27h1v1H18zM23 27h2v1H23zM28 27h3v1H28zM0 28h1v1H0zM2 28h3v1H2zM6 28h1v1H6zM9 28h2v1H9zM13 28h4v1H13zM18 28h1v1H18zM20 28h2v1H20zM24 28h5v1H24zM30 28h2v1H30zM0 29h1v1H0zM2 29h3v1H2zM6 29h1v1H6zM10 29h1v1H10zM12 29h7v1H12zM21 29h2v1H21zM24 29h1v1H24zM28 29h2v1H28zM31 29h1v1H31zM0 30h1v1H0zM2 30h3v1H2zM6 30h1v1H6zM9 30h1v1H9zM11 30h1v1H11zM13 30h1v1H13zM15 30h3v1H15zM22 30h1v1H22zM28 30h1v1H28zM30,30 h3v1H30zM0 31h1v1H0zM6 31h1v1H6zM9 31h1v1H9zM11 31h3v1H11zM23 31h1v1H23zM25 31h1v1H25zM30 31h1v1H30zM0 32h7v1H0zM9 32h6v1H9zM18 32h2v1H18zM21 32h1v1H21zM24 32h1v1H24zM28 32h1v1H28zM30 32h2v1H30z"
```

Comment:
are these path changes innocuous?

Reply:
It's because of the changes to the election hash.